Docs update 2.5.0 stabilization (#8512)
microsoft/MixedRealityToolkit-Unity: Documentation/WelcomeToMRTK.md

---
title: Welcome to MRTK
description: Landing page for the MRTK-Unity documentation site.
author: hferrone
ms.author: v-hferrone
ms.date: 09/9/2020
ms.localizationpriority: high
keywords: Unity,HoloLens, HoloLens 2, Mixed Reality, development, MRTK,
---

# Welcome to MRTK

![MRTK Logo](../Documentation/Images/MRTK_Logo_Rev.png)

The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed Reality experiences for Virtual Reality (VR) and Augmented Reality (AR). 

## Development 

If you're new to MRTK or Mixed Reality development in Unity, we recommend you start at the beginning of our [Unity development journey](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview?tabs=mrtk%2Chl2). The Unity development journey is the **recommended starting point for MRTK**, specifically created to walk you through the installation, core concepts, and usage of MRTK in Unity. 

> [!CAUTION]

> > The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.3.15**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).

Reviewer: We should probably recommend 2019.4 over 2019.3.

```suggestion
> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

> > The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.3.15**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).

Reviewer: We should probably recommend Unity 2019.4 over 2019.3.

```suggestion
> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

Reviewer: Do we also want to reference 2.5?

```suggestion
> The Unity development journey currently uses **MRTK version 2.5.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

> ## Documentation Versioning

Reviewer: ```suggestion
## Documentation versioning
```
